dvc/command/imp_url.py

```diff
@@ -14,7 +14,10 @@ class CmdImportUrl(CmdBase):
     def run(self):
         try:
             self.repo.imp_url(
-                self.args.url, out=self.args.out, fname=self.args.file
```

Comment:
if we change `imp_url` we should probably change `import` as well?

Reply:
make sense :+1: